docs/data_sources/eia860.rst

```diff
@@ -11,94 +11,103 @@ Source Description  | The status of existing electric generating plants and
 Respondents         Utilities
 Source Format       Microsoft Excel (.xls/.xlsx)
 Source Years        2001-2019
-Size (Download)     127 MB
-Size (Uncompressed) 247 MB
+Size (Download)     XX MB
+Size (Uncompressed) XX MB
 PUDL Code           ``eia860``
-Years Liberated     2009-2018
-Records Liberated   ~600,000
+Years Liberated     2004-2019
+Records Liberated
 Issues              `open EIA 860 issues <https://github.com/catalyst-cooperative/pudl/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3Aeia860>`__
 =================== ===========================================================
 
 Background
 ^^^^^^^^^^
 
-The Form EIA-860 collects environmental equipment and generator-level data from
-extant as well as planned generation entities with one  or more megawatts of
-capacity. Beginning in 2013,  EIA-860 also collects construction cost data.
+The Form EIA-860 collects utility, owner, plant, and generator-level data from existing
+and planned entities with one or more megawatt of capacity. The form also contains
+information regarding environmental control equipment, and in 2013, began collecting
+construction cost data.
 
+As of 2019, the EIA-860 Form is organized into the following schedules:
 
-**Who is required to fill out the form?**
+* **Schedule 1:** Identification
+* **Schedule 2:** Power plant data
+* **Schedule 3:** Generator information
+* **Schedule 4:** Ownership of generators owned jointly or by others
+* **Schedule 5:** Generator construction cost information
```

Comment:
yup, you're right! the last year was 2018

Reply:
I do wonder if we should try and provide multi-year perspective on these things when we can though, since we're trying to cover all available years.  I've been adding historical instructions / forms to the documentation for reference.